lib/OpenQA/CacheService.pm

```diff
@@ -76,7 +76,7 @@ sub startup {
     $sqlite->on(
         connection => sub {
             my ($sqlite, $dbh) = @_;
-            my $sqlite_mode = uc($ENV{OPENQA_CACHE_SERVICE_SQLITE_JOURNAL_MODE} || 'WAL');
+            my $sqlite_mode = uc($ENV{OPENQA_CACHE_SERVICE_SQLITE_JOURNAL_MODE} || 'DELETE');
```

Comment:
could you please describe here a little bit why you chose DELETE and what are the implications of WAL? E.g. describe what this is fixing but what are the differences to the other modes. Something like a hint to users that consider changing the mode with env variable.